docs/docs/guides/automate/declarative-automation/index.md

```diff
@@ -1,88 +1,96 @@
-| -------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------------------------- |
-| `AutomationCondition.on_cron(cron_schedule)` | This condition will materialize an asset on a provided `cron_schedule`, after all of its dependencies have been updated.                                                                                                                               | Regularly updating an asset without worrying about the specifics of how its dependencies update.         |
-| `AutomationCondition.on_missing()`           | This condition will materialize an asset if all its dependencies have been updated, but the asset itself has not.                                                                                                                                      | Filling in partitioned assets as soon as upstream data is available.                                     |
-| `AutomationCondition.eager()`                | This condition will materialize an asset: <ul><li>If the asset has never been materialized before, or</li><li>When the asset's dependencies update, as long as none of the dependencies are currently missing or have an update in progress.</li></ul> | Automatically propagating changes through the asset graph.<br /><br />Ensuring assets remain up to date. |
+- If at least one upstream partition of _all_ upstream assets has been updated since the previous cron schedule tick, and the downstream asset has not yet been requested or updated, it will be kicked off
+- If all upstream assets **do not** update within the given cron tick, the downstream asset will not be kicked off
+- For **time-partitioned** assets, this condition will only kick off the _latest_ time partition of the asset
+- For **static** and **dynamic-partitioned** assets, this condition will kick off _all_ partitions of the asset
 
-### Setting automation conditions on assets and asset checks
+</TabItem>
 
-You can set automation conditions on the <PyObject section="assets" module="dagster" object="asset" decorator /> decorator or on an <PyObject section="assets" module="dagster" object="AssetSpec" /> object:
+  <TabItem value="on_missing" label="on_missing">
 
-```python
-import dagster as dg
+The <PyObject section="assets" module="dagster" object="AutomationCondition.on_missing" /> condition will execute an asset when all upstream partitions of the asset have been filled in. This is typically used to manage dependencies between partitioned assets, where partitions should fill in as soon as upstream data is available.
```

Comment:
I know that we talk about partitioned assets here, but consider adding something like:

```
The `on_missing` condition will execute a missing partitioned asset when the upstream dependencies for that partition are available.
```

The `manage dependencies between partitioned assets` doesn't hit home for me... I can have a `on_missing` for a partitioned asset whose upstream assets are all unpartitioned, right?